app/move/controllers/new.form.js

```diff
@@ -10,6 +10,16 @@ class FormController extends Controller {
     this.use(this.checkCurrentLocation)
   }
 
+  middlewareLocals() {
+    super.middlewareLocals()
+    this.use(this.setCancelUrl)
+  }
+
+  setCancelUrl(req, res, next) {
+    res.locals.cancelUrl = `/moves${res.locals.MOVES_QUERY_STRING || ''}`
```

Comment:
See my [previous comment](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/pull/177#discussion_r310991322) I think setting the full path in the middleware would avoid duplication here.

Reply:
Will address once the other PR has been merged.